src/Uno.UI/MixinGeneration.targets

```diff
@@ -2,9 +2,6 @@
 <Project>
 
   <PropertyGroup>
-	<T4Path>$(MSBuildThisFileDirectory)..\T4Generator\bin\$(Configuration)</T4Path>
-	<T4Bin>$(T4Path)\T4Generator.exe</T4Bin>
```

Comment:
I don't think having an external dependency will make it easy on developers to work with Uno. We should be validating the presence of the tool being installed as follows:
https://github.com/unoplatform/uno/blob/e2248d2a89f089915e1c5c93ddf4aab98ef671bd/build/test-scripts/wasm-run-automated-uitests.sh#L13

Let's install it in obj (Interdemediate path) to avoid git detection.